projects/1.19/assets/fabric-seasons-extras/seasonsextras/lang/zh_cn.json

```
@@ -0,0 +1,121 @@
+{
+  "itemGroup.seasonsextras.creative_tab": "Fabric Seasons：拓展",
+
+  "item.seasonsextras.seasonal_compendium": "季节汇编",
+  "item.seasonsextras.crop_season_tester": "作物季节检测仪",
+
+  "block.seasonsextras.season_calendar": "季节历",
+  "block.seasonsextras.season_detector": "季节探测器",
+  "block.seasonsextras.black_greenhouse_glass": "黑色温室玻璃",
+  "block.seasonsextras.blue_greenhouse_glass": "蓝色温室玻璃",
+  "block.seasonsextras.brown_greenhouse_glass": "棕色温室玻璃",
+  "block.seasonsextras.cyan_greenhouse_glass": "青色温室玻璃",
+  "block.seasonsextras.gray_greenhouse_glass": "灰色温室玻璃",
+  "block.seasonsextras.green_greenhouse_glass": "绿色温室玻璃",
+  "block.seasonsextras.light_blue_greenhouse_glass": "淡蓝色温室玻璃",
+  "block.seasonsextras.light_gray_greenhouse_glass": "淡灰色温室玻璃",
+  "block.seasonsextras.lime_greenhouse_glass": "黄绿色温室玻璃",
+  "block.seasonsextras.magenta_greenhouse_glass": "品红色温室玻璃",
+  "block.seasonsextras.orange_greenhouse_glass": "橙色温室玻璃",
+  "block.seasonsextras.pink_greenhouse_glass": "粉红色温室玻璃",
+  "block.seasonsextras.purple_greenhouse_glass": "紫色温室玻璃",
+  "block.seasonsextras.red_greenhouse_glass": "红色温室玻璃",
+  "block.seasonsextras.white_greenhouse_glass": "白色温室玻璃",
+  "block.seasonsextras.yellow_greenhouse_glass": "黄色温室玻璃",
+  "block.seasonsextras.tinted_greenhouse_glass": "着色温室玻璃",
+  "block.seasonsextras.heater": "作物加热器",
+  "block.seasonsextras.chiller": "作物冷却器",
+
+  "tooltip.seasonsextras.calendar_info_1": "当前为",
+  "tooltip.seasonsextras.calendar_info_2": "剩余天数到",
```

推测这行前面放天数，后面放季节
**本条需要实测**
```suggestion
  "tooltip.seasonsextras.calendar_info_2": "天以后到",
```

---

是这样的（